.github/workflows/tests.yaml

```diff
@@ -8,68 +8,18 @@ on:
     branches:
       - "ubuntu-*"
   schedule:
-    # run at 00:00 on the first of every month
-    - cron: "0 0 1 * *"
+    # Run at 00:00 every day.
+    # Ref: https://man7.org/linux/man-pages/man5/crontab.5.html
+    - cron: "0 0 * * *"
 
 jobs:
-  planner:
-    name: Determine jobs
-    runs-on: ubuntu-latest
-    outputs:
-      # install-all is set to 'true' if chisel.yaml changes
-      install-all: ${{ steps.changed-paths.outputs.chisel-yaml }}
-      # install-changed is set to 'true' if any slice definition files
-      # have been added or modified (not deleted).
-      install-changed: ${{ steps.changed-paths.outputs.slices }}
-      # changed-sdf is a space separated list of slice definition file
-      # paths who have been added or modified (not deleted).
-      changed-sdf: ${{ steps.changed-paths.outputs.slices_files }}
-    steps:
-      - uses: actions/checkout@v4
-        if: ${{ github.event_name != 'schedule' }}
-        with:
-          fetch-depth: 0
-
-      - name: Check changed paths
-        id: changed-paths
-        if: ${{ github.event_name != 'schedule' }}
-        uses: dorny/paths-filter@v3
-        with:
-          # ref: https://github.com/marketplace/actions/paths-changes-filter
-          filters: |
-            chisel-yaml:
-              - 'chisel.yaml'
-            slices:
-              - added|modified: 'slices/**/*.yaml'
-          # Space delimited list usable as command-line argument list in
-          # Linux shell. If needed, it uses single or double quotes to
-          # wrap filename with unsafe characters.
-          list-files: shell
-
-  install-all:
-    name: Install all slices
-    needs: planner
-    if: ${{ needs.planner.outputs.install-all == 'true' }}
-    strategy:
-      fail-fast: false
-      matrix:
-        ARCH: [amd64, arm64, armhf, i386, ppc64el, riscv64, s390x]
-    uses: canonical/chisel-releases/.github/workflows/install-slices.yaml@main
-    with:
-      arch: ${{ matrix.ARCH }}
-      install-all: true
-
-  install-changed:
-    name: Install slices in changed files
-    needs: planner
-    if: |
-      needs.planner.outputs.install-all != 'true' &&
-      needs.planner.outputs.install-changed == 'true'
+  installability:
+    name: Installability
     strategy:
       fail-fast: false
       matrix:
         ARCH: [amd64, arm64, armhf, i386, ppc64el, riscv64, s390x]
```

Comment:
this matrix can also be on `main`

Reply:
Done! https://github.com/canonical/chisel-releases/pull/120/commits/5b22495a138e38c97c6783986a013e3a4d915bc2